### PR TITLE
[home] Added image banner for Expo home.

### PR DESCRIPTION
**Pull request by EvanBacon**
# Why

Utilize the Camera, MediaLibrary, and associated permissions in the home app. 

# How

Added a banner image to the user profile which is stored in local state. Then using the MediaLibrary, and Cameraja code from NCL, integrated a system for choosing images.

# Test Plan

Should be able to run home in home with expo start. Go to your user profile and add a banner image by pressing the "Change Banner" button. If you reset the app, the banner should persist. Other profiles will show a default image until actually integrating with a server sometime in the future.

IMG_0363
IMG_0362
IMG_0361

### CODE REVIEW

File: home/screens/MediaLibrary/MediaDetailsScreen.tsx [hunk 1]
```diff
@@ -0,0 +1,158 @@
...
+      this.goBack();
+    }
+  };
+
+  renderAsset(asset: MediaLibrary.Asset) {
+    const aspectRatio = asset.height ? asset.width / asset.height : 1;
+
+    switch (asset.mediaType) {
+      case MediaLibrary.MediaType.photo:
+      case MediaLibrary.MediaType.video: // TODO: render Expo.Video component
```
Comment (brentvatne) [inline home/screens/MediaLibrary/MediaDetailsScreen.tsx]: what happens in the current form if someone chooses a videO?

File: home/screens/MediaLibrary/MediaAlbumsScreen.tsx [hunk 2]
```diff
@@ -0,0 +1,146 @@
...
+  Platform,
+  StyleSheet,
+  Switch,
+  Text,
+  TouchableOpacity,
+  View,
+} from 'react-native';
+import { NavigationScreenProps } from 'react-navigation';
+
+// import MonoText from '../../components/MonoText';
```
Comment (ide) [inline home/screens/MediaLibrary/MediaAlbumsScreen.tsx]: fix whatever is going on here

File: home/screens/MediaLibrary/MediaDetailsScreen.tsx [hunk 3]
```diff
@@ -0,0 +1,158 @@
+import * as MediaLibrary from 'expo-media-library';
```
Comment (ide) [inline home/screens/MediaLibrary/MediaDetailsScreen.tsx]: This whole screen looks irrelevant

File: home/screens/MediaLibrary/MediaLibraryScreen.tsx [hunk 4]
```diff
@@ -0,0 +1,323 @@
...
+export default class MediaLibraryScreen extends React.Component<NavigationScreenProps, State> {
+  static navigationOptions: NavigationScreenConfig<{}> = ({ navigation }) => {
+    const goToAlbums = () => navigation.navigate('MediaAlbums');
+    const clearAlbumSelection = () => navigation.setParams({ album: null });
+    const { params } = navigation.state;
+    const isAlbumSet = params && params.album;
+
+    return {
+      title: 'Choose a photo',
+      // headerRight: (
```
Comment (ide) [inline home/screens/MediaLibrary/MediaLibraryScreen.tsx]: remove

File: home/screens/MediaLibrary/MediaLibraryScreen.tsx [hunk 5]
```diff
@@ -0,0 +1,323 @@
...
+
+  onEndReached = () => {
+    this.loadMoreAssets();
+  };
+
+  onCellPress = (asset: MediaLibrary.Asset) => {
+    console.log('chosen: ', asset);
+    Store.dispatch(ProfileActions.setImage(asset.uri));
+    this.props.navigation.goBack();
+    // this.props.navigation.navigate('MediaDetails', {
```
Comment (ide) [inline home/screens/MediaLibrary/MediaLibraryScreen.tsx]: remove

File: home/screens/MediaLibrary/MediaLibraryScreen.tsx [hunk 6]
```diff
@@ -0,0 +1,323 @@
...
+  };
+
+  keyExtractor = (item: MediaLibrary.Asset) => item.id;
+
+  onEndReached = () => {
+    this.loadMoreAssets();
+  };
+
+  onCellPress = (asset: MediaLibrary.Asset) => {
+    console.log('chosen: ', asset);
```
Comment (ide) [inline home/screens/MediaLibrary/MediaLibraryScreen.tsx]: remove

**Review by ide — CHANGES_REQUESTED** [review]
Rebase after your kernel and options buttons commits, and also clean up unused cruft.

File: home/screens/MediaLibrary/MediaAlbumsScreen.tsx [hunk 7]
```diff
@@ -0,0 +1,143 @@
...
+import { NavigationScreenProps } from 'react-navigation';
+
+interface State {
+  includeSmartAlbums: boolean;
+  albums: MediaLibrary.Album[];
+}
+
+export default class MediaAlbumsScreen extends React.Component<NavigationScreenProps, State> {
+  static navigationOptions = {
+    title: 'MediaLibrary Albums',
```
Comment (ide) [inline home/screens/MediaLibrary/MediaAlbumsScreen.tsx]: This doesn't look right..

**Pull request merged**